### DISCUSSION TITLE
How to add contour?

### DISCUSSION THREAD

**Discussion by 1923trk** [dlemstra/Magick.NET#1658] [Help]
Hello, Thank you for the library.
I have png images with alpha channels. I want to add contours to these images and get a result like the examples I will share below, but I couldn't find how to do it. Additionally I should be able to control the background color.
![54](https://github.com/kleisauke/net-vips/assets/172930587/db260337-be94-4e5d-90d5-5bed5cebdd19)
![24061513YKHV-01cqkdy8yiiputopi8le3xut](https://github.com/kleisauke/net-vips/assets/172930587/83f391d7-bfd7-4246-9453-89e29c4bce5d)
![169](https://github.com/kleisauke/net-vips/assets/172930587/77d32b28-24c6-47c9-9687-e9b82f3d3319)
![24061513YKGS-t7qy707avbs1bx3p82webwwq](https://github.com/kleisauke/net-vips/assets/172930587/816d2eba-2342-4835-898c-e73e8601150a)

**Comment by dlemstra** [comment]
There is no function to do that and I am also not sure if that is even possible. Maybe you could ask your question here instead: https://github.com/ImageMagick/ImageMagick/discussions. And if you get an answer and need help translation this to C# then feel free to come back and ask for more help.
